bedrock/firefox/templates/firefox/channel/base.html

```diff
@@ -46,18 +46,19 @@ <h2 class="c-sub-navigation-title">
 
   {% block channels %}{% endblock %}
 
-  {% call picto_card(
-    title=ftl('firefox-channel-see-something-that-isnt-working'),
+<ul class="mzp-t-picto-center">
+  {% call picto(
+    image_url='img/icons/warning.svg',
     heading_level=2,
-    class='bugzilla',
-    base_el='section',
-    custom_desc=True) %}
+    image_width=44,
+    title=ftl('firefox-channel-see-something-that-isnt-working'),
+    body=True,
+    ) %}
 
-  <ul>
```

Comment:
We need this opening `<ul>` tag for the list items below, and the closing `</ul>` tag should come before the `{% endcall %}` so whole list is inside the picto body.